lib/shopify-cli/tunnel.rb

```diff
@@ -141,6 +141,13 @@ def install(ctx)
         ctx.rm(zip_dest)
       end
       spinner.wait
+
+      # final check to see if ngrok is accessible
+      file_name = ctx.windows? ? 'ngrok.exe' : 'ngrok'
+      unless File.exist?(File.join(ShopifyCli.cache_dir, file_name))
+        raise "Something went wrong with ngrok installation, "\
```

Comment:
It might be cleaner to use `abort` to make the output cleaner – ideally end users shouldn't be exposed a stack trace.